pages/bookmarks.jsx

```diff
@@ -49,10 +49,10 @@ export default React.createClass({
     return(<p>Hi {user.username}! <button onClick={this.handleLogOutBtnClick} className="btn btn-link inline-link">Log out</button>.</p>);
   },
   getFailurePrompt() {
-    return (<p>Only Mozilla staff can login now as we test this new platform. Meanwhile, your favorited projects can still be remembered on this device.</p>);
+    return (<p>Something went wrong! Please refresh and try again.</p>);
```

Comment:
@Sudip7 – Right now we only have google login and we don't have specific plans to change that at this time. No need to worry about that now. Thanks!